java-frontend/src/main/java/org/sonar/java/se/FlowComputation.java

```diff
@@ -254,15 +269,17 @@ public boolean hasAlwaysSameConstraint(@Nullable Symbol symbol) {
     final PSet<ExplodedGraph.Edge> visited;
     final Flow flow;
     final boolean finished;
+    final boolean skipExceptionMessages;
```

Comment:
You don't need field in `ExecutionPath`, it never changes during graph traversal, you can use field from `FlowComputation`